api/src/main/java/org/pmiops/workbench/api/ConceptSetsController.java

```diff
@@ -44,6 +44,14 @@ public class ConceptSetsController implements ConceptSetsApiDelegate {
     this.userProvider = userProvider;
   }
 
+  @Override
+  public ResponseEntity<Integer> countConceptsInConceptSet(
```

Comment:
Removing the concept set hydration raised a need for a new endpoint to get the count of concepts per specific concept set.